packages/terra-graphs/CHANGELOG.md

```diff
@@ -23,5 +23,5 @@
 * Changed
   * Made changes to WDIO test url's according to terra rules. 
   * Updated components to maintain consistent design.
-
+  * Updated Pie chart snapshots.
```

Comment:
nit: newline after this

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/217/files#diff-d24190ff244a9f2416eeb3cbe315e41345dab81295d0fcccd748cd7f6c93a84aR27).